TESTING.md

```diff
@@ -14,22 +14,22 @@ This library provides tools to help write tests for code that uses the following
 You can test against a temporary local datastore by following these steps:
 
 1. Start the local datastore emulator using `LocalGcdHelper`.  This can be done in two ways:
-  - Run `LocalGcdHelper.java`'s `main` method with arguments `START` and (optionally) `--port=<port number>`.  This will create a temporary folder on your computer and bind `localhost:<port number>` for communication with the local datastore.  The port number is an optional argument.  If no port number is specified, port 8080 will be used.
-  - Call `LocalGcdHelper.start(<project ID>, <port number>)` before running your tests.  Save the `LocalGcdHelper` object returned so that you can stop the emulator later.
-
-2. In your program, create and use a datastore whose host is set host to `localhost:<port number>`.  For example, 
+  - Run `LocalGcdHelper.java`'s `main` method with `START` provided as an argument, followed by optional arguments `--port=<port number>` and `--consistency=<float between 0 and 1, inclusive>`.  This will create a temporary folder on your computer and bind `localhost:<port number>` for communication with the local datastore.  If no port number is specified, port 8080 will be used.  The consistency setting controls the fraction of Datastore writes that are immediately visible in global queries.
+  - Use the `LocalGcdHelper.start(String projectId, int port, double consistency)` method before running your tests. For example, you can use the following code to start the local Datastore on any available port with the consistency set to 0.9:
+    ```java
+    int port = LocalGcdHelper.findAvailablePort(LocalGcdHelper.DEFAULT_PORT);
```

Comment:
Any reason for not going all the way and automatically using port selection (and remove the static methods), so one should keep a reference to the same instance of the helper and stop it once tests complete.  At some point I was using the main method to start (and also calling from maven) but I think by now this is long gone, no?